docs/namespaced-broker-resources.md

```diff
@@ -0,0 +1,159 @@
+
+```console
+helm install svc-cat/catalog \
+   --name catalog \
+   --namespace catalog \
+   --set namespacedServiceBrokerEnabled=true
+```
+
+If you are also using RBAC, providing this option to the Helm install command will also create the needed RBAC resources.
+
+## Using Namespace Scoped Broker Resources
+
+Once Service Catalog has been installed with this feature gate enabled, you should see three new resource types: `ServiceBroker`, `ServiceClass`, and `ServicePlan`.
+
+In order to register a `ServiceBroker` resource, you create a YAML definition that looks similar to a `ClusterServiceBroker`. This resource will use a different resource kind, however, and requires a namespace. An example might look like:
+
+```yaml
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceBroker
+metadata:
+  name: example-ns-broker
+  namespace: ns-broker
+spec:
+  authInfo:
+    basic:
+      secretRef:
+        name: my-service-broker-auth
+        namespace: broker
+  url: http://my-service-broker.broker.svc.cluster.local
+```
+
+Once this resource is created, Service Catalog will query the Service Broker for the list of available Services and create corresponding `ServiceClass` and `ServicePlan` resources. These resources might look like this:
+
+```yaml
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceClass
+metadata:
+  creationTimestamp: 2018-07-12T13:30:01Z
+  name: 25434f16-d762-41c7-bbdd-8045d7f74ca6
+  namespace: ns-broker
+  resourceVersion: "13"
+  selfLink: /apis/servicecatalog.k8s.io/v1beta1/namespaces/ns-broker/serviceclasses/25434f16-d762-41c7-bbdd-8045d7f74ca6
+  uid: adfa2d9a-85d7-11e8-a4f3-2ae408f4a9e4
+spec:
+  bindable: true
+  bindingRetrievable: false
+  description: MySQL
+  externalID: 25434f16-d762-41c7-bbdd-8045d7f74ca6
+  externalName: mysql-5-7
+  planUpdatable: false
+  serviceBrokerName: example-ns-broker
+status:
+  removedFromBrokerCatalog: false
+```
+
+```yaml
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServicePlan
+metadata:
+  creationTimestamp: 2018-07-12T13:30:02Z
+  name: 4c6932e8-30ec-4af9-83d2-6e27286dbab3
+  namespace: ns-broker
+  resourceVersion: "24"
+  selfLink: /apis/servicecatalog.k8s.io/v1beta1/namespaces/ns-broker/serviceplans/4c6932e8-30ec-4af9-83d2-6e27286dbab3
+  uid: ae8e23ac-85d7-11e8-a4f3-2ae408f4a9e4
+spec:
+  description: basic plan
+  externalID: 4c6932e8-30ec-4af9-83d2-6e27286dbab3
+serviceBrokerName: example-ns-broker
+  serviceClassRef:
+    name: 25434f16-d762-41c7-bbdd-8045d7f74ca6
+status:
+  removedFromBrokerCatalog: false
+```
+
+The `ServiceInstance` resource has also been updated to allow you to use these resources just as you would the existing `ClusterServiceBroker`, `ClusterServiceClass` and `ClusterServicePlan` resources, except you will use them in the context of a namespace. For example, a `ServiceInstance` YAML that references a `ClusterServiceClass` and a `ClusterServicePlan` resource might look like this:
```

Comment:
+1, this passage is really helpful to illustrate the differences, especially for those already familiar with the existing cluster-scoped behavior.